pow/src/main/java/tech/pegasys/teku/pow/Eth1DepositManager.java

```diff
@@ -165,12 +168,11 @@ private void preGenesisSubscription(final BigInteger lastProcessedBlockNumber) {
   }
 
   private SafeFuture<EthBlock.Block> getHead() {
-    return eth1Provider
-        .getGuaranteedLatestEth1Block()
-        .thenApply(EthBlock.Block::getNumber)
-        .thenApply(UInt64::valueOf)
-        .thenCompose(this::waitForEth1ChainToReachFollowDistanceIfNecessary)
-        .thenApply(number -> number.minus(Constants.ETH1_FOLLOW_DISTANCE))
+    final SafeFuture<UInt64> headBlockNumber = new SafeFuture<>();
```

Comment:
nit: we're now a future, maybe rename variable?

Reply:
Done.